PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -2090,28 +2090,6 @@ class PurchasesTests: XCTestCase {
         expect(completionCalled).toEventually(beTrue())
     }
 
-    func testAddsDiscountToWrapper() {
-        if #available(iOS 12.2, tvOS 12.2, macOS 10.14.4, *) {
-            setupPurchases()
-            let product = MockSK1Product(mockProductIdentifier: "com.product.id1")
-            let discount = SKPaymentDiscount(
-                identifier: "discount",
-                keyIdentifier: "TIKAMASALA1",
-                nonce: UUID(),
-                signature: "Base64 encoded signature",
-                timestamp: NSNumber(value: Int64(123413232131))
-            )
-
-            self.purchases?.purchase(product: product, discount: discount) { (_, _, _, _) in
-
-            }
-
-            expect(self.storeKitWrapper.payment).toNot(beNil())
-            expect(self.storeKitWrapper.payment?.productIdentifier).to(equal(product.productIdentifier))
-            expect(self.storeKitWrapper.payment?.paymentDiscount).to(equal(discount))
-        }
-    }
```

Comment:
This test did not seem like it was needed anymore since `StoreKitWrapper` is no longer being called in `Purchases`.

With this change, `StoreKitWrapper` is now being used in `PurchasesOrchestrator` so I will be adding new tests for that

Reply:
> StoreKitWrapper is no longer being called in Purchases.

That's a good sign!